javascript/javascript1/week4/Ali-Nematollahi-Homework/square-digits.js

```diff
@@ -0,0 +1,9 @@
+function squareDigits(num) {
+  const numString = num.toString();
+  resultNumStr = "";
+  for (i in numString) {
```

Comment:
```suggestion
  for (let i in numString) {
```
Same comment as the other file.